### ISSUE TITLE
Expired login tokens should redirect to ONID login

### ISSUE THREAD

**Issue by JacksGo** [osu-capstone-cs72/cs-applied-plan-portal#76]
I just ran into this about five minutes ago while finishing up the refactor:
If you keep a session open until the login expires, you'll eventually try to navigate to a page and receive a 404 error. This isn't because the page has been deleted, but rather because the user has effectively logged out, and the API seems to default to responding to unauthorized requests with 404 instead of 403. Especially in the context of a plan page, this is extremely nerve wracking and should be fixed by redirecting unauthorized page requests to the ONID login.

**Comment by silverware13** [comment]
Sure. I will look into this.

**Comment by JacksGo** [comment]
On hold. It's possible the server died on me, because I now can't start my local build with the same missing cookie error as before.
